tests/kernel/testPointVector.cpp

```diff
@@ -357,12 +357,12 @@ TEST_CASE( "4D Point Vector Unit tests" )
       REQUIRE( p1.dot(p2) == 30 );
       REQUIRE( dotProduct(p1, p2) == 30 );
 
-      REQUIRE( p1.cosineSimilarity(p1) == Approx(0.) );
-      REQUIRE( p1.cosineSimilarity(-p1) == Approx(pi) );
-      REQUIRE( p1.cosineSimilarity( Point(-2,1,-4,3) ) == Approx(pi/2) );
-      REQUIRE( cosineSimilarity(p1, p1) == Approx(0.) );
-      REQUIRE( cosineSimilarity(p1, -p1) == Approx(pi) );
-      REQUIRE( cosineSimilarity(p1, Point(-2,1,-4,3)) == Approx(pi/2) );
+      REQUIRE( p1.cosineSimilarity(p1) == Approx(0.).margin(0.00001));
```

Comment:
`0.000001` instead ? (and following lines)

Reply:
Done.
Thx for the commits and comments.